### ISSUE TITLE
Cannot load the dll when packaged

### ISSUE THREAD

**Issue by jimsimonz** [AleDel/Spout-UE4#30]
Although the spout.dll is placed in Plugins\Spout\ThirdParty\Spout\lib\amd64 when packaged it will not load from here and is seen as a missing dependency.

The only way to get it to load is to move the dll to the game root folder.

**Comment by SeanJD** [comment]
Hi, I'm also having this issue (though it doesn't even seem to load if I place in the game root folder?) - whenever the spout plugin is used it throws a Fatal Error! crash window. It's fine when playing in editor but not after packaging (Win 64) - did you find out what was causing this?

**Comment by jimsimonz** [comment]
@SeanJD Nope, I've found the same issue/crash for receiving Spout streams in packaged games.

Looking at the code it's a bit C like, everything's static so needs a bit of OOD applying.

My app uses sending spout streams only and is fine packaged.

**Comment by visiblegroup** [comment]
I have the same issue - need to copy Spout.dll to game folder/Binaries.

@SeanJD if you receive async loading fatal error after launching the Packaged game you need to package it without using .pak files. Check one of the closed issues of this repo.
